### PR TITLE
Features2

### PR DESCRIPTION
**Pull request by leolpaz**
In this project, I attempt to make a proper responsive copy of The Next Web homepage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,828 @@
...
+    </nav>
+
+  </header>
+
+  <main>
+    <div class="box">
+
+    </div>
+    <section class="top-news">
+      <div class="top-news1">
```
Comment (umairarshadbutt) [inline index.html]: Use the `article` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,828 @@
...
+        </div>
+        <div class="bottom-tittle tittle">
+          Facebook’s ‘supreme court’ overturns most of the platform’s post removals in its first rulings
+        </div>
+        <div class="author">
+          by Thomas Macaulay <span><i class=" fas fa-clock"></i> 2</span> hours ago
+          <span><i class="fas fa-share-alt"></i> 0</span>
+        </div>
+      </div>
+      <div class="top-news2">
```
Comment (umairarshadbutt) [inline index.html]: Use the `article` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,828 @@
...
+            Tech
+          </div>
+          Why cybersecurity needs an API-first mentality
+        </div>
+        <div class="author">
+          by Patrick Coughlin <span><i class=" fas fa-clock"></i> 3</span> hours ago <span><i
+              class="fas fa-share-alt"></i> 10</span>
+        </div>
+      </div>
+      <div class="top-news3">
```
Comment (umairarshadbutt) [inline index.html]: Use the `article` tag

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @leolpaz 
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
#### Top 3 articles section
- [x] Please use `atricle` tag for these section

#### Latest deals section
- [x] Use the percenatge off ribin on deals.
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @leolpaz, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏   I appreciate your hard work in replicating thenextweb.com. Kindly fix the following issues. 

[REQUIRED]
- [ ] Remove unnecessary and copied(LICENSE copy, and if u have others) files from the project.
- [ ] The latest new section right side components should have the same height as the left side bigger one in the medium screens.

| Your Copy | Source |
| :---: | :---: |
| Screen Shot 2021-01-30 at 5 25 39 AM|Screen Shot 2021-01-30 at 5 24 12 AM|

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by layzrd — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @leolpaz, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  Congratulations 🎊 👏, you did very well on this project.

Now, you can move ahead.

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Pull request merged**